### PR TITLE
feat(driving-license-application): Health check remarks on fullnaðarskírteini

### PR DESCRIPTION
**Pull request by alexdiljar**
If user had health remarks on bráðabirgðaskírteini then they will be shown in application for fullnaðarskírteini.
Fake data for testing 
Screenshot 2022-05-04 at 13 34 54
Screenshot 2022-05-04 at 13 35 13

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/driving-license/src/forms/prerequisites/sectionFakeData.ts [hunk 1]
```diff
@@ -81,6 +82,31 @@ export const sectionFakeData = buildSubSection({
...
+            )
+            return (
+              fakeData?.currentLicense === 'temp' &&
+              fakeData.useFakeData === YES
+            )
+          },
+          options: [
+            {
+              value: NO,
+              label: 'Enging tákntala',
```
Comment (albinagu) [inline libs/application/templates/driving-license/src/forms/prerequisites/sectionFakeData.ts:102]: Engin* and move to messages pls

**Review by kksteini — APPROVED** [review]

File: libs/application/templates/driving-license/src/fields/HealthRemarks.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  const remarks: string[] =
+    getValueViaPath<CurrentLicenseProviderResult>(
+      application.externalData,
+      'currentLicense.data',
+    )?.healthRemarks || []
+
+  const { setValue } = useFormContext()
+
+  useEffect(() => {
+    setValue('hasHealthRemarks', remarks?.length > 0 ? 'yes' : 'no')
```
Comment (albinagu) [inline libs/application/templates/driving-license/src/fields/HealthRemarks.tsx:25]: YES and NO constants maybe?

File: libs/application/templates/driving-license/src/forms/prerequisites/sectionFakeData.ts [hunk 3]
```diff
@@ -81,6 +82,31 @@ export const sectionFakeData = buildSubSection({
             },
           ],
         }),
+        buildRadioField({
+          id: 'fakeData.healthRemarks',
+          title: 'Heilbrigðis tákntala',
```
Comment (albinagu) [inline libs/application/templates/driving-license/src/forms/prerequisites/sectionFakeData.ts:87]: msgs

**Review by albinagu — APPROVED** [review]
👍

**Pull request merged**